taprpc/taprootassets.proto

```diff
@@ -627,6 +627,10 @@ message AssetTransfer {
 
     // Describes the set of newly created asset outputs.
     repeated TransferOutput outputs = 6;
+
+    // The block hash the contains the anchor transaction above. If unset, the
+    // anchor transaction is unconfirmed.
+    bytes anchor_tx_block_hash = 7;
```

Comment:
Same request as here: https://github.com/lightninglabs/taproot-assets/pull/1075#discussion_r1713581013
Whenever we reference block or transaction hashes, I think we should also return the human-readable (byte-reversed) variant that can be copy/pasted easily from the command line. So just add an additional `anchor_tx_block_hash_str` that returns the `.String()` variant.

Reply:
OK, I'll add this in a future PR (you mention in the review that you're ok with that). I think I've got a good pattern in mind.